api/MediaKeyStatusMap.json

```diff
@@ -109,10 +109,10 @@
               "version_added": "13"
             },
             "firefox": {
-              "version_added": null
+              "version_added": "47"
```

Comment:
This looks a bit suspect, that `forEach` should have been added later than `@@iterator`, since both are part of `iterable<T>` in Web IDL. Unfortunately https://github.com/mozilla/gecko-dev/blame/master/dom/webidl/MediaKeyStatusMap.webidl did not have any clues. Can you double check what happened here manually?

Reply:
I've gone ahead and tested http://mdn-bcd-collector.appspot.com/tests/api/MediaKeyStatusMap in Firefox 46 and 47 and now can't doubt these results. All subfeatures except this test true in Firefox 46, so it really must have been missing. And `MediaKeyStatusMap` doesn't inherit from anything, so we don't need to test an instance to rule out that it was just moved around.